astropy/visualization/mpl_normalize.py

```diff
@@ -226,3 +228,61 @@ def simple_norm(data, stretch='linear', power=1.0, asinh_a=0.1, min_cut=None,
     vmin, vmax = interval.get_limits(data)
 
     return ImageNormalize(vmin=vmin, vmax=vmax, stretch=stretch, clip=clip)
+
+
+# used in imshow_norm
+_norm_sig = inspect.signature(ImageNormalize)
+
+
+def imshow_norm(data, ax=None, **kwargs):
+    """ A convenience function to call matplotlib's `matplotlib.pyplot.imshow`
+    function, using an `ImageNormalize` object as the normalization.
+
+    Parameters
+    ----------
+    data
+        The data to show. Can be whatever type `matplotlib.pyplot.imshow` and
+        `ImageNormalize` accept.
+    ax : None or `~matplotlib.axes.Axes`
+        If None, use pyplot's imshow.  Otherwise, calls ``imshow`` method of the
+        supplied axes.
+
+    All other keyword arguments are parsed first by the `ImageNormalize`
+    initializer, then to ``imshow``.
+
+    Notes
+    -----
+    The ``data`` matplotlib keyword is not supported.
```

Comment:
It's not clear what you mean here (I assume you mean `imshow` keyword).  But -- is this note even necessary?  Perhaps I missed something.

Reply:
Oops!  Typo - that was supposed to be the ``norm`` keyword, since that would just override `ImageNormalize` completely.